src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+public class Ui {
+
+    public String showError(String error) {
+        System.out.println(error);
+        System.out.println("");
+        return error + "\n";
+    }
+
+    public String showAddedTask(Task task, int numOfTasks) {
+        return "Got it. I've added this task:\n" + task
+                + "\nNow you have " + numOfTasks + " tasks in the list. \n";
+    }
+
+    public String showRemovedTask(Task task, int numOfTasks) {
```

Comment:
I like the intuitive naming of all the functions which makes it easy to understand